src/utils/ApiImport.ts

```diff
@@ -1,6 +1,27 @@
+import { DefaultState, useState } from "~/State";
+import { useCharacters } from "~/composables/Characters";
 import Ajv from "ajv";
 import apiSchema from "~/utils/ApiSchema.json";
 
+type ImportData = Partial<typeof DefaultState>;
+
 const ajv = new Ajv();
+// TODO: decide on how we want to do semantic versioning for the API schema
```

Comment:
Will this be an additional field in the JSON or just a change on your end?

Reply:
Per our discussion in Discord, it will probably be a separate field from the version embedded in the JSON. We need some way to semantically validate the JSON is up to date, preferably with a single field like this rather than checking all the fields programmatically. I also think for a prototype/until the schema changes, we don't really need to worry about this.

Reply:
Yep, that makes sense to me. I like the idea of keeping the game version separate from the API version. Thanks for clarifying.